simulator/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

```diff
@@ -53,12 +53,13 @@
     <push-ros-namespace namespace="perception"/>
     <!-- object segmentation module -->
     <group>
-      <push-ros-namespace namespace="obstacle_segmentation"/>
+      <push-ros-namespace namespace="occupancy_grid_map"/>
       <!-- Occupancy Grid -->
       <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
         <arg name="input_obstacle_pointcloud" value="true" />
         <arg name="input_obstacle_and_raw_pointcloud" value="false" />
         <arg name="input/obstacle_pointcloud" value="/perception/obstacle_segmentation/pointcloud" />
```

Comment:
@yukkysaito 
Note :　this topic in perception launch is `/perception/obstacle_segmentation/single_frame/pointcloud_raw`

Reply:
Although in Psim ground segmentation is unnecessary so `/perception/obstacle_segmentation/pointcloud` stays still